### ISSUE TITLE
Try to avoid intermediate conversion to Healpix grid

### ISSUE THREAD

**Issue by mreineck** [yusuke-takase/levels_beam#5]
The step of going to a Healpix grid before converting to `a_lm` should probably be avoided for accuracy reasons. The original `beam2alm` code did actually produce a Healpix map, but it was only used for diagnostic purposes; the actual SHTs were performed on an equidistant (theta, phi) grid using the function `beam_transform%bm_polar2alm`.

I'll try to provide the necessary `ducc0.sht` calls in a pull request soon.

**Comment by yusuke-takase** [comment]
Thank you for the information. The fortran module [`beam_transform`](https://github.com/zonca/planck-levelS/blob/master/Beam/beam_transform.f90) seems to be worked if there is the function that corresponds `sharpf_do_job_d` as you say. After the PR I can work on the fortran translation to python!

**Comment by mreineck** [comment]
Yes, I have exactly the required function in `ducc0`, and I plan to integrate it directly into my PR, so hopefully no extra work  should be necessary.

Actually, maybe I even have a way to extract `a_lm` directly from the rectangular grid of a GRD beam. That would be very handy, but I cannot promise this yet...
